modules/ztp-creating-the-policygentemplate-cr.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-creating-the-policygentemplate-cr_{context}"]
+= Creating the PolicyGenTemplate CR
+
+Use this procedure to create the `PolicyGenTemplate` custom resource (CR) for your
+site in your local clone of the Git repository.
+
+.Procedure
+
+. Choose an appropriate example from `out/argocd/example/policygentemplates`.
+This directory demonstrates a three-level policy framework that represents a well-supported low-latency
+profile tuned for the needs of 5G Telco DU deployments:
++
+* A single `common-ranGen.yaml` file that should apply to all types of sites.
+* A set of shared `group-du-*-ranGen.yaml` files, each of which should be common across a
+set of similar clusters.
+* An example `example-*-site.yaml` that can be copied and updated for each individual site.
+
+. Ensure that the labels defined in your `PolicyGenTemplate` `bindingRules` section correspond to the labels that are defined in the `SiteConfig` files of the clusters you are managing.
```

Comment:
This is correct. However, without the context given in the examples shown later in the document under the "Deploying a site" heading, it may not make sense to an end user reading the document from top to bottom. Can you link to the example SiteConfig later in the document for context?

Reply:
@StephenJamesSmith  do you mean the link to another piece of doc can't be linked at this time, but will be added in final draft?

Reply:
@yliu127 The build machine we currently use does no allow internal links in modules. We've been told that this functionality will be added sometime in the future. Sorry.

Reply:
I see. Thanks for clarifying.